chapter_preliminaries/probability.md

```diff
@@ -1,304 +1,751 @@
 
 ### Random Variables
 
-In our experiment of casting a die, we introduced the notion of a *random variable*. A random variable can be pretty much any quantity. It can take on a value among a set of (possibly many) possibilities.
-Consider a random variable $X$ whose value is in the sample space $\mathcal{S} = \{1, 2, 3, 4, 5, 6\}$ of rolling a die. We can denote the event "seeing a $5$" as $\mathcal{A} := \{X = 5\}$ or $X = 5$, and its probability as $P(\{X = 5\})$ or $P(X = 5)$.
-
-By $P(X = a)$, we make a distinction between the random variable $X$ and the values (e.g., $a$) that $X$ can take.
-However, such pedantry results in a cumbersome notation and we omit it unless necessary.
-On the one hand, we can just denote $P(X)$ as the *distribution* over the random variable $X$:
-the distribution tells us the probability that $X$ takes any value.
-On the other hand,
-we can simply write $P(a)$ to denote the probability that a random variable takes the value $a$.
+When we spoke about events like the roll of a die
+coming up odds or the first coin toss coming up heads,
+we were invoking the idea of a *random variable*.
+Formally, random variables are mappings
+from an underlying sample space 
+to a set of (possibly many) values.
+You might wonder how a random variable 
+is different from the sample space,
+since both are collections of outcomes. 
+Importantly, random variables can be much coarser 
+than the raw sample space.
+We can define a binary random variable like "greater than 0.5"
+even when the underlying sample space is infinite,
+e.g., the line segment between $0$ and $1$.
+Additionally, multiple random variables
+can share the same underlying sample space.
+For example "whether my home alarm goes off"
+and "whether my house was burglarized"
+are both binary random variables 
+that we share an underlying sample space. 
+Consequently, knowing the value taken by one random variable 
+can tell us something about the likely value of another random variable.
+Knowing that the alarm went off, 
+we might suspect that the house was likely burglarized.
+
+
+Every value taken by a random variable corresponds 
+to a subset of the underlying sample space.
+Thus the occurrence where the random variable $X$
+takes value $v$, denoted by $X=v$, is an *event*
+and $P(X=v)$ denotes its probability.
+Sometimes this notation can get clunky,
+and we can abuse notation when the context is clear.
+For example, we might use *P(X)* to refer broadly
+to the *distribution* of $X$, i.e., 
+the function that tells us the probability
+that $X$ takes any given value. 
+Other times we write expressions 
+like $P(X,Y) = P(X) P(Y)$,
+as a shorthand to express a statement 
+that is true for all of the values
+that the random variables $X$ and $Y$ can take, i.e.,
+for all $i,j$ $P(X=i \textrm{ and } Y=j) = P(X=i)P(Y=j)$.
```

Comment:
Though this is grammatically sound, we may further improve readability if we insert "we have/it holds that" between $i,j$ and $P()$, otherwise it may look to beginners that:
for all i,
j P(xxx) = P(xx)P(xx)

Reply:
don't understand what you mean.